apk-generator/v2/app/utils/notification.py

```diff
@@ -46,7 +78,23 @@ def send_mail_via_smtp_(config, payload):
         mailer.start()
         message = Message(author=payload['from'], to=payload['to'])
         message.subject = payload['subject']
-        message.plain = strip_tags(payload['html'])
-        message.rich = payload['html']
+        message.plain = payload['message']
```

Comment:
The `message` is a rich text string. (Has html tags). So, have both `message.plain` (with html tags stripped) and `message.rich`